SPECS/maven/maven.spec

```diff
@@ -73,7 +66,7 @@ popd
 # Changing distribution dir to BUILD directory as install macro clears buildroot prior to creating a fresh directory, thus clearing artifacts copied by maven. We copy them later.
 MAVEN_DIST_DIR=%{_builddir}%{_prefixmvn}
 
-export JAVA_HOME="%{_libdir}/jvm/msopenjdk-11"
+export JAVA_HOME=$(find %{_libdir}/jvm -name "msopenjdk*")
```

Comment:
Do we want to keep this change?  Seems like the explicit -11 was/is correct?

Reply:
Yes, JAVA_HOME should be explicitly set to point to msopenjdk-11 during build. I will upate.

Reply:
Done.